eegexpy/__init__.py

```diff
@@ -0,0 +1,7 @@
+
+from eegnb.experiments import VisualN170,VisualP300,VisualSSVEP,AuditoryOddball
+
+from eegnb.analysis.utils import *
+
+
+
```

Comment:
You should specify a `__all__` variable here with the intended "exports" of the package (what will be imported with `from eegexpy import *`).

Reply:
Can you elaborate? 

I had hoped the `import *` would give me all of the `eegnb` imports and that would be the only line needed

In practice that didn't work. Unclear why. From reading it seemed it could be that the modules inside `eegnb` are doing relative imports. So it looks like I will need to explicitly do a load of imports in this init to get it to mirror `eegnb`